packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -159,7 +159,7 @@ var getDate = function (dateStringRaw) {
     console.log(
       `[getDate] Date was an object (converted it to string): ${JSON.stringify(dateStringRaw)}`
     );
-    dateStringRaw = dateStringRaw.toString();
+    dateStringRaw = dateStringRaw.value ? dateStringRaw.value.toString() : dateStringRaw.toString();
```

Comment:
Should we only log in case the object doesn't have `.value`? Then we don't trash the logs and still learn about other formats we want to add fix for.